runtime-common/core/array_access.h

```diff
@@ -0,0 +1,24 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+#include "common/php-functions.h"
+#include "runtime-common/core/class-instance/refcountable-php-classes.h"
+
+#ifndef INCLUDED_FROM_KPHP_CORE
+  #error "this file must be included only from runtime-core.h"
+#endif
+
+struct C$ArrayAccess : public may_be_mixed_base {
+  virtual int get_hash() const noexcept = 0;
+
+  C$ArrayAccess() __attribute__((always_inline)) = default;
```

Comment:
Let's not use these attributes. These methods are implicitly inline

Reply:
Firstly, this attribute to about linkage things, it's about forcing compiler to inline this function in terms of getting rid of extra function call.
But this is exactly how it would be codegenerated and I want to leave consistency with codegen.

Reply:
I mean that any definition inside class declaration is implicitly `inline`, so no need to add explicit attributes here

Reply:
Also, you can add `noexcept` here.